api_tests/README.rst

```diff
@@ -0,0 +1,34 @@
+API tests
+#########
+
+Functional and safety tests against the API of a running instance.
```

Comment:
Can we mention that these are not part of pytest, and either need to be run manually or could be a part of automated testing as part of a deployment. You do not intend to run these as part of CI tests, right?

Reply:
Yeah, I'll add some sort of note to that effect.